### PR TITLE
chore(cf-exports): rely on CF's exports - EUBFR-131

### PR DESCRIPTION
**Pull request by yhuard**
To test:

- remove all the `.serverless`
- try to deploy the whole stack

### CODE REVIEW

File: demo/dashboard/server/serverless.yml [hunk 1]
```diff
@@ -11,10 +11,10 @@ custom:
   EUBFR_USERNAME: ${opt:username, env:EUBFR_USERNAME, file(../../../config.json):username, ''}
   PRODUCER_KEY_ID: ${opt:PRODUCER_KEY_ID, file(../../../config.json):demo.${self:custom.EUBFR_USERNAME}.AWS_ACCESS_KEY_ID, env:PRODUCER_KEY_ID, ''}
   PRODUCER_SECRET_ACCESS_KEY: ${opt:PRODUCER_SECRET_ACCESS_KEY, file(../../../config.json):demo.${self:custom.EUBFR_USERNAME}.AWS_SECRET_ACCESS_KEY, env:PRODUCER_SECRET_ACCESS_KEY, ''}
-  SIGNED_UPLOADS_API: ${opt:SIGNED_UPLOADS_API, file(../../../services/storage/signed-uploads/.serverless/stack-output.json):ServiceEndpoint, env:SIGNED_UPLOADS_API, ''}
-  DELETER_API: ${opt:DELETER_API, file(../../../services/storage/deleter/.serverless/stack-output.json):ServiceEndpoint, env:DELETER_API, ''}
-  output:
-    file: .serverless/stack-output.json
+  SIGNED_UPLOADS_API:
```
Comment (kalinchernev) [inline demo/dashboard/server/serverless.yml]: Code using these env vars needs to be updated, previously these values contained `https://`.

File: demo/dashboard/server/serverless.yml [hunk 2]
```diff
@@ -11,10 +11,10 @@ custom:
...
+    Fn::ImportValue: ${self:provider.stage}:storage-signed-uploads:ServiceEndpoint
```
Comment (kalinchernev) [inline demo/dashboard/server/serverless.yml]: This one also needs `https://` in client.

**Review by kalinchernev — CHANGES_REQUESTED** [review]

File: demo/dashboard/server/serverless.yml (hunk 2 above)
Comment (yhuard) [reply demo/dashboard/server/serverless.yml]: Thanks, will fix it 👍

**Review by kalinchernev — APPROVED** [review]

**Comment by kalinchernev** [comment]
Thanks!

**Pull request merged**